### PR TITLE
feat: 🎸 Hooked up instrumentation

### PR DESCRIPTION
**Pull request by JohanObrink**
https://trello.com/c/f3Gt4cYO

Performance of individual crypto actions can now be seen in APM (in Kibana)

### CODE REVIEW

File: services/data.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const read = async (accessToken) => {
 export const write = async (accessToken, data, area) => {
   await axios.post(`/api/data/${area}`, data[area], {
     headers: {
-      'Authorization': 'Bearer ' + accessToken,
+      Authorization: 'Bearer ' + accessToken,
       'Content-Type': 'application/json'
```
Comment (iteadam) [inline services/data.js:16]: Remove ' on content-type as well

File: pages/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Box, Typography, Button, theme } from '@smooth-ui/core-sc'
 
 export default () => {
   return (
-    <React.Fragment>
+    <>
```
Comment (iteadam) [inline pages/index.js:6]: Do we really need this empty element?

File: pages/_app.js [hunk 3]
```diff
@@ -46,18 +44,16 @@ export default class MyDataCV extends App {
...
-          <StoreProvider>
-            <Navbar />
-            <Component {...pageProps} />
-          </StoreProvider>
-        </Container>
-      </React.Fragment>
+        <StoreProvider>
+          <Navbar />
+          <Component {...pageProps} />
+        </StoreProvider>
```
Comment (iteadam) [inline pages/_app.js:55]: I'm not sure what these changes are about, but it works :man_shrugging:

**Review by iteadam — APPROVED** [review]
~~Pls fix the minor issues I found before merging~~
Haha lol they were not issues it was just me who was silly everything is fine :+1:

File: services/data.js (hunk 1 above)
Comment (JohanObrink) [reply services/data.js:16]: LOL! U suck! Like a black hole.

Comment (iteadam) [reply services/data.js:16]: Just liek ur mom

Comment (JohanObrink) [reply services/data.js:16]: She pulls

Comment (iteadam) [reply services/data.js:16]: I force push

Comment (JohanObrink) [reply services/data.js:16]: Teh -f awakens

File: pages/index.js (hunk 2 above)
Comment (iteadam) [reply pages/index.js:6]: Yes, this is a fragment

**Pull request merged**